requirements.in

```diff
@@ -7,7 +7,7 @@ attrs<21
 # https://github.com/flyteorg/flyte/issues/1732.
 jsonschema<4
 pyyaml<6
-# A number of dependencies, including pandas and numpy, started dropping support for python 3.7 in their latest
-# releases (pandas>=1.4.0 and numpy>=1.22.0). More details in https://github.com/flyteorg/flyte/issues/2115.
+# A number of dependencies, including pandas and NumPy, started dropping support for Python 3.7 in their latest
```

Comment:
Is the version # right?

Reply:
This isn't something I am sure about, since I only changed the word from 'numpy' to 'NumPy'.

Reply:
Let it be.